src/Console/Command/FixCommand.php

```diff
@@ -388,6 +388,6 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
     protected function isDryRun(InputInterface $input): bool
     {
-        return $input->getOption('dry-run');
+        return $input->getOption('dry-run'); // @phpstan-ignore symfonyConsole.optionNotFound (Because PHPStan doesn't recognise the method is overwritten in child class and this parameter is _not_ in use in child class.)
```

Comment:
```suggestion
        return $input->getOption('dry-run'); // @phpstan-ignore symfonyConsole.optionNotFound (Because PHPStan doesn't recognise the method is overridden in the child class and this parameter is _not_ used in the child class.)
```